scripts/start_e2eaiok_docker.py

```diff
@@ -295,25 +295,29 @@ def main(input_args):
 
     hostname = os.uname()[1]
     print_success = False
-    if input_args.backend == 'pytorch':
-        docker_name = "e2eaiok-pytorch110"
-        docker_file = "DockerfilePytorch110"
-        docker_nickname = "e2eaiok-pytorch"
-        port = 12345
+    # latest tensorflow env
     if input_args.backend == 'tensorflow':
-        docker_name = "e2eaiok-tensorflow210"
-        docker_file = "DockerfileTensorflow210"
+        docker_name = "e2eaiok-tensorflow"
+        docker_file = "DockerfileTensorflow"
         docker_nickname = "e2eaiok-tensorflow"
         port = 12344
-    if input_args.backend == 'pytorch_mlperf':
+    # pytorch1.10 env
+    if input_args.backend == 'pytorch110':
+        docker_name = "e2eaiok-pytorch110"
+        docker_file = "DockerfilePytorch110"
+        docker_nickname = "e2eaiok-pytorch110"
+        port = 12345
+    # pytorch1.5 env
+    if input_args.backend == 'pytorch':
         docker_name = "e2eaiok-pytorch"
         docker_file = "DockerfilePytorch"
-        docker_nickname = "e2eaiok-pytorch-mlperf"
+        docker_nickname = "e2eaiok-pytorch"
         port = 12346
-    if input_args.backend == 'tensorflow205':
-        docker_name = "e2eaiok-tensorflow"
-        docker_file = "DockerfileTensorflow"
-        docker_nickname = "e2eaiok-tensorflow205"
+    # pytorch1.12 env
+    if input_args.backend == 'pytorch120':
```

Comment:
if pytorch120 is runnable for v0.2 model, you can use pytorch as backend name for pytprch120
Please update readmes for rnnt to use port 12347